components/activity/issuer.go

```diff
@@ -1,54 +1,22 @@
 package activity
 
 import (
-	"github.com/iotaledger/hive.go/lo"
-	"github.com/iotaledger/iota-core/pkg/model"
 	iotago "github.com/iotaledger/iota.go/v4"
-	"github.com/iotaledger/iota.go/v4/builder"
 )
 
-func issuerID() iotago.AccountID {
-	issuerKey := lo.PanicOnErr(deps.Peer.Database().LocalPrivateKey())
-	pubKey := issuerKey.Public()
-
-	return iotago.AccountID(iotago.Ed25519AddressFromPubKey(pubKey[:]))
-}
-
 func issueActivityBlock() {
 	if !ParamsActivity.IgnoreBootstrapped && !deps.Protocol.MainEngineInstance().IsBootstrapped() {
 		Component.LogDebug("Not issuing activity block because node is not bootstrapped yet.")
 		return
 	}
```

Comment:
Shouldn't this check be part of the issuer package?

Reply:
I think the caller should assess if the protocol is bootstrapped and what wants to do with it. It would have convoluted the activity for instance, as we have to ignore this bootstrapped value in some situation, but at the same time you don't want to ignore it throughout the blockissuing.